aptos-move/framework/aptos-framework/sources/Account.move

```diff
@@ -343,18 +343,70 @@ module AptosFramework::Account {
     }
 
     ///////////////////////////////////////////////////////////////////////////
-    /// Basic account creation method.
+    /// Basic account creation methods.
     ///////////////////////////////////////////////////////////////////////////
 
     public(script) fun create_account(auth_key: address) {
-        let (signer, _) = create_account_internal(auth_key);
+        let signer = create_account_internal(auth_key);
         Coin::register<TestCoin>(&signer);
     }
 
+    public fun create_module_account(
```

Comment:
This function is misleading - create_module_account can actually be called to create accounts that do not need to host modules. They can be used as normal accounts. I think we should consider restricting the scope of this to just modules. Being able to freely create accounts that users don't have keys for that are not for modules only can have wider security and trust implications that we might not be able to fully anticipate.

Reply:
I don't disagree, but the problem is that we'd have to make some substantial changes to the adapter or the account resource to limit that and it isn't entirely clear to me what the implications would be here. What are the problems we want to solve?

Given that we need to start stimulating the community to build, I think we should keep this in mind, but move forward.